cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnamolyDetectionSuite.scala

```diff
@@ -73,38 +71,39 @@ class FitMultivariateAnomalySuite extends EstimatorFuzzing[FitMultivariateAnomal
     .setInputCols(inputColumns)
 
   test("Blob client SAS Url generation is correct") {
-    val blobName = "intermediateData/some_file.zip"
+    val blobName = "intermediateData/some_file.csv"
     val offset = OffsetDateTime.parse("2022-08-22T18:52:02.853001800-04:00")
     val exampleSas = "sv=2020-10-02&se=2022-08-22T22%3A52%3A02Z&sr=b&sp=r&" +
-      "sig=QIl7NpjUhsWHXy5tt3WynqHSNEuhbS0qLmEzI9gUfa0%3D"
+      "sig=ECIwvPFBjiB4HHz%2B4z3D8nEomDtZZ9B7vAiLiT7VACg%3D"
 
     val sas1 = SlimStorageClient.generateReadSAS(
```

Comment:
Do we need this sas and it’s corrresponding generator anymore or can we delete it all?

Reply:
We don't need it anymore because they don't support SAS url anymore. But do we want to keep it just in case for future usage?

Reply:
You can remove and we'll point people to git history if needed